src/System/Requirement/InstallationNotOverriden.php

```diff
@@ -67,6 +75,19 @@ protected function check()
             return;
         }
 
+        $file_iterator->rewind();
+        $file_version = $file_iterator->current()->getFilename();
+        $current_version = preg_replace('/^(\d+\.\d+\.\d+)(-.+)?$/', '$1', GLPI_VERSION);
```

Comment:
You could reuse existing version parser method.
```suggestion
        $current_version = \Glpi\Toolbox\VersionParser::getNormalizedVersion(GLPI_VERSION, false);
```